tabula/io.py

```diff
@@ -1,6 +1,6 @@
-"""This module is a wrapper of tabula, which enables extract tables from PDF.
+"""This module is a wrapper of tabula, which enables the extraction of tables from a PDF.
```

Comment:
Maybe this line causes lint error. Can you check your local environment?

Or, this would be okay:
```suggestion
"""This module is a wrapper of tabula, which enables table extraction from a PDF.
```

Reply:
```suggestion
"""
This module is a wrapper of tabula, which enables table extraction from a PDF.
```
This resolves the linting error that my contribution evokes.

Reply:
Also, on passing the entire tabula-py project (without my contribution) through flake8, I found the following linting errors were already present:
![flake8_errors](https://user-images.githubusercontent.com/15872393/90331955-4ad9bb00-dfd6-11ea-867e-858211755f3b.png)